src/platform/pic32/uart.cpp

```diff
@@ -26,6 +26,10 @@
 #include "atcommander.h"
 #include "WProgram.h"
 #include "gpio.h"
+#include "util/timer.h"
+
+#warning "mikeG add some plib"
```

Comment:
Shouldn't have been left behind. I thought I needed to include plib to get access to the UART registers but I did not. Removing on next commit.